OracleDatabase/SingleInstance/dockerfiles/23.2.0/runOracle.sh

```diff
@@ -0,0 +1,343 @@
+#!/bin/bash
+# LICENSE UPL 1.0
+#
+# Copyright (c) 1982-2023 Oracle and/or its affiliates. All rights reserved.
+# 
+# Since: November, 2016
+# Author: gerald.venzl@oracle.com
+# Description: Runs the Oracle Database inside the container
+# 
+# DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS HEADER.
+# 
+
+########### Move DB files ############
+function moveFiles {
+
+   if [ ! -d "$ORACLE_BASE"/oradata/dbconfig/"$ORACLE_SID" ]; then
+      mkdir -p "$ORACLE_BASE"/oradata/dbconfig/"$ORACLE_SID"/
+   fi;
+
+   mv "$ORACLE_BASE_CONFIG"/dbs/spfile"$ORACLE_SID".ora "$ORACLE_BASE"/oradata/dbconfig/"$ORACLE_SID"/
+   mv "$ORACLE_BASE_CONFIG"/dbs/orapw"$ORACLE_SID" "$ORACLE_BASE"/oradata/dbconfig/"$ORACLE_SID"/
+   mv "$ORACLE_HOME"/network/admin/sqlnet.ora "$ORACLE_BASE"/oradata/dbconfig/"$ORACLE_SID"/
+   mv "$ORACLE_HOME"/network/admin/listener.ora "$ORACLE_BASE"/oradata/dbconfig/"$ORACLE_SID"/
+   mv "$ORACLE_HOME"/network/admin/tnsnames.ora "$ORACLE_BASE"/oradata/dbconfig/"$ORACLE_SID"/
+   # shellcheck disable=SC2144
+   if [ -a "$ORACLE_HOME"/install/.docker_* ]; then
```

Comment:
This shellcheck should not be disabled. This if statement will fail if there are more than 1 `.docker_*` files present:

```
gvenzl$ touch .docker_1
gvenzl$ touch .docker_2
gvenzl$ touch .docker_3
gvenzl$ if [ -a .docker_* ]; then
>   echo true;
> fi;
-bash: [: too many arguments
gvenzl$
```

Reply:
resolving this with 
```
for file in "$ORACLE_HOME"/install/.docker_*
   do 
      if [ -e "$file" ]; then 
         mv "$file" "$ORACLE_BASE"/oradata/dbconfig/"$ORACLE_SID"/
      fi 
   done
```

Reply:
Better replaced the for loop with find command alternative
`find "$ORACLE_HOME"/install/ -name ".docker_*"  -exec  mv {} "$ORACLE_BASE"/oradata/dbconfig/"$ORACLE_SID"/  \;`

Reply:
Awesome, that's even better!